src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  src: url();
+}
+
+body {
+  min-width: 950px;
+  max-width: 1300px;
+  margin: 0 auto;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+}
+
+.logo {
+  flex-grow: 1;
+  margin: 10px 0 10px 50px;
+}
+
+.title {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0,0,0,0) !important;
+  white-space: nowrap !important;
+}
```

Comment:
don't use important

Reply:
I added this snippet for hide title h1. It is for semantic